app-cdr/cdrtools/cdrtools-3.01_alpha17-r1.ebuild

```diff
@@ -0,0 +1,172 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit fcaps multilib toolchain-funcs flag-o-matic gnuconfig
+
+MY_P="${P/_alpha/a}"
```

Comment:
This is more a matter of choice, but I prefer `versionator` over bash substitutions. Consider this optional though.